stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/attachments/content/FileAttachmentContent.kt

```diff
@@ -217,6 +225,6 @@ public fun FileAttachmentImage(attachment: Attachment) {
         modifier = imageModifier,
         painter = painter,
         contentDescription = null,
-        contentScale = if (isImage) ContentScale.Crop else ContentScale.Fit
+        contentScale = if (isImage || isVideoWithThumbnails) ContentScale.Crop else ContentScale.Fit
```

Comment:
I did not add the play button here. 
File content items are really small and the play button would be difficult to see here.

Let me know if you think that's the wrong decision.

Reply:
Sounds good to me - people can always open each file to preview it individually, so it should be okay